src/mainwindow.cpp

```diff
@@ -308,12 +308,12 @@ void MainWindow::showConsoleContextMenu(const QPoint &pt)
 
 void MainWindow::closeEvent(QCloseEvent *event)
 {
-    QMessageBox::StandardButton ret = QMessageBox::question(this, "Iaito",
+    QMessageBox::StandardButton ret = QMessageBox::question(this, "iaito",
                  "Do you really want to exit?\nSave your project before closing!",
                  QMessageBox::Ok | QMessageBox::Cancel);
     //qDebug() << ret;
     if (ret == QMessageBox::Ok) {
-        QSettings settings("Radare Corp", "Iaito");
+        QSettings settings("iaito", "iaito");
```

Comment:
First value sets the directory name, which should be "iaito/".
Any specific reason why they should be different?
Not sure what to call the file, since Qt puts the ".conf" suffix on there.

In the long run you might want to have different config files for system/Iaito-set values and user-set values, like vimrc/vimrc.local.